### PR TITLE
Closes #778: Migration for D7 Jumbotron paragraph type (to AZ Text)

### LINKED ISSUE #778: Migrate Jumbotron paragraph type
See migration mapping doc for details: https://docs.google.com/spreadsheets/d/134UMiYtE8f1YAKYgSukL2JxFzDJF6-TgCAvYFC-vxqQ/edit#gid=0

### PR DESCRIPTION
**Pull request by smitahantzios**
Migrate Jumbotron paragraph type

## Description
Migrate Jumbotron paragraph type

## Related Issue

Closes #778 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I've added this Pull Request to the AZ Quickstart Github project and set it to "Review in progress".
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: modules/custom/az_migration/src/Plugin/migrate/process/ParagraphsJumbotronFieldMerge.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    $value['uaqs_text'] = '<div class="jumbotron">';
+    if ($field_uaqs_short_title[0]['value'] != "") {
+      $value['uaqs_text'] .= '<h1 class="display-3 mt-0">' . $field_uaqs_short_title[0]['value'] . '</h1>';
+    }
+    if ($field_uaqs_summary[0]['value'] != "") {
+      $value['uaqs_text'] .= '<div class="lead">' . $field_uaqs_summary[0]['value'] . '</div>';
+    }
+    if ($field_uaqs_links[0]['url']) {
+      $value['uaqs_text'] .= '<a href="' . $field_uaqs_links[0]['url'] . '" class="' . $field_uaqs_links[0]['attributes']['class'] . '">' . $field_uaqs_links[0]['title'] . '</a>';
+    }
```
Comment (tadean) [inline modules/custom/az_migration/src/Plugin/migrate/process/ParagraphsJumbotronFieldMerge.php:35]: I think it'd be safer to use `!empty` for these comparisons.

**Review by danahertzberg — APPROVED** [review]
Looks like all requirements are met from a content perspective

File: modules/custom/az_migration/src/Plugin/migrate/process/ParagraphsJumbotronFieldMerge.php (hunk 1 above)
Comment (smitahantzios) [reply modules/custom/az_migration/src/Plugin/migrate/process/ParagraphsJumbotronFieldMerge.php]: Update the PR with condition.

File: modules/custom/az_migration/src/Plugin/migrate/process/ParagraphsJumbotronFieldMerge.php [hunk 2]
```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Drupal\az_migration\Plugin\migrate\process;
+
+use Drupal\migrate\MigrateExecutableInterface;
+use Drupal\migrate\ProcessPluginBase;
+use Drupal\migrate\Row;
+
+/**
+ * Configure Behavior for paragraphs.
```
Comment (tadean) [inline modules/custom/az_migration/src/Plugin/migrate/process/ParagraphsJumbotronFieldMerge.php:10]: This class comment appears to need updating for this file.

**Review by danahertzberg — APPROVED** [review]

**Comment by danahertzberg** [comment]
Fix Troy's suggestion, then ready to merge when all checks pass.

File: modules/custom/az_migration/src/Plugin/migrate/process/ParagraphsJumbotronFieldMerge.php (hunk 2 above)
Comment (smitahantzios) [reply modules/custom/az_migration/src/Plugin/migrate/process/ParagraphsJumbotronFieldMerge.php]: Done

**Review by danahertzberg — APPROVED** [review]

**Review by joeparsons — APPROVED** [review]
This one looks good to me now.

**Review by tadean — APPROVED** [review]

**Pull request merged**